modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -3846,11 +3846,6 @@ public void testGetObjectEntryWithAuditEvents() throws Exception {
 			PermissionThreadLocal.setPermissionChecker(
 				originalPermissionChecker);
 		}
-
-		_objectDefinitionLocalService.deleteObjectDefinition(objectDefinition);
-
-		_listTypeDefinitionLocalService.deleteListTypeDefinition(
-			listTypeDefinition);
```

Comment:
Why these two deletions?
Maybe I missed something on the review but from what I've seen, the objectDefinition and the listTypeDefinition are created on the test and needs to be deleted here as they are not deleted on the `tearDown`

Without deleting it we could leave some trash on the DB.

Am I correct or I missed something from the refactors?

Reply:
I checked without the deletions, and the elements are being deleted properly. I deleted because with that, the test was failing (I guess because the object definition is deleted twice, one in the code and one with the test rules)

Reply:
Finally removed the commit as it is not failing locally in my local, so better not to change anything